impl/src/rendering/templates/partials/crates_macro.template

```diff
@@ -0,0 +1,183 @@
+# A mapping of package names to a set of normal dependencies for the Rust targets of that package.
```

Comment:
For each of these maps, does it make sense to include a description of the key and value?

Reply:
The maps should all be the same but contain targets of differing significant. So explaining one should be sufficient. Also, I touched on this in another comment. I felt these maps should be private and users should be made to go through the interface macros `crates` and `all_crates` but I didn't want to be too restrictive.

Reply:
They've been made private